tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -1,28 +1,28 @@
 #!/bin/bash
 set -e
-while getopts p:d: parameter_Option
+while getopts p: parameter_Option
```

Comment:
looks like you removed the d option accidentally?

Reply:
I merged the script changes from branch jywu_trt6. Will take a closer look at the scripts.

Reply:
> I merged the script changes from branch jywu_trt6. Will take a closer look at the scripts.

thanks. that branch wasn't meant for merging directly.